app/components/polaris/index_table_component.html.erb

```diff
@@ -1,21 +1,25 @@
 <%= render Polaris::BaseComponent.new(**system_arguments) do %>
-  <div class="Polaris-IndexTable-ScrollContainer">
-    <table class="Polaris-IndexTable__Table">
-      <thead>
-        <tr>
-          <% columns.each_with_index do |column, index| %>
-            <th class="Polaris-IndexTable__TableHeading">
-              <%= column.title %>
-            </th>
-          <% end %>
-        </tr>
-      </thead>
-      <tbody>
-        <% @data.each do |row| %>
-          <%= render Polaris::BaseComponent.new(**row_arguments(row)) do %>
+  <% if @data.empty? %>
```

Comment:
I am too lazy to do this everytime on my app's side

Reply:
Perfect motivation to improve it 🙂